source/MRCommonPlugins/ViewerButtons/MRIOFilesMenuItems.cpp

```diff
@@ -73,7 +73,7 @@ OpenFilesMenuItem::OpenFilesMenuItem() :
         setupListUpdate_();
         connect( &getViewerInstance() );
         // required to be deferred, for valid emscripten static constructors oreder 
-        filters_ = MeshLoad::getFilters() | LinesLoad::Filters | PointsLoad::Filters | SceneFileFilters | DistanceMapLoad::Filters;
+        filters_ = MeshLoad::getFilters() | LinesLoad::Filters | PointsLoad::Filters | SceneFileFilters | DistanceMapLoad::Filters | VoxelsLoad::Filters;
```

Comment:
VoxelFilters are not supported in __EMSCRIPTEN__

Reply:
fixed